ipa-core/src/helpers/gateway/mod.rs

```diff
@@ -98,6 +102,7 @@ impl Gateway {
         mpc_transport: MpcTransportImpl,
         shard_transport: ShardTransportImpl,
     ) -> Self {
+        tracing::info!("active_work = {}", config.active);
```

Comment:
maybe `debug`?

Reply:
The reason I made it `info` is that there is logic in `helpers::transport::query` that configures the active work dynamically to be no more than the query input size. At the time I didn't see the note in the commit description "Input size constraint comes from our CI that is so slow, it cannot allow very large active work sizes.", and the logic didn't make sense to me -- but I figured that if we really do need to be tuning it adaptively per query, then we probably also want to log it per query, at a level that we don't need to restart the server in debug mode to see.

But now that I have seen the commit message, it makes more sense. I will add the comment about CI in the code though.